### PR TITLE
SPL-462 - Create privacy policy

### PR DESCRIPTION
**Pull request by Louis-Solirius**
[SPL-462](https://solirius.atlassian.net/browse/SPL-462)

### CODE REVIEW

File: app/views/privacy-notice.njk [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+{% endblock %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds-from-desktop">
+            <h1 class="govuk-heading-xl">
+                Privacy notice
+            </h1>
+
+            <p>This privacy notice explains how the Department for Business and Trade (DBT), as a ‘data controller’, processes personal data for the ‘Check if you can get Shared Parental Leave and Pay’ service on GOV.UK. </p>
```
Comment (devind-ra) [inline app/views/privacy-notice.njk:28]: Add  ‘Plan your Shared Parental Leave and Pay’ for ‘Check if you can get Shared Parental Leave and Pay’

File: app/views/privacy-notice.njk [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            <p>In this service, DBT collects information about individuals planning Shared Parental Leave and their family members. This service does not collect any personal data that allows the direct identification of an individual (for example names).</p>
+            <p>The following categories of personal data are collected:</p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>Dates of birth</li>
+                <li>Salary and employment information</li>
+                <li>Details of a child’s birth, adoption or surrogacy</li>
+            </ul>
+
+            <h2 class="govuk-heading-m">Why DBT asks for this information and what happens if it is not provided</h2>
+            <p>Personal data is collected through the ‘Check if you can get Shared Parental Leave and Pay’ service to enable individuals to check their eligibility for Shared Parental Leave and Shared Parental Pay. This is an optional service that individuals are not required to use.</p>
```
Comment (devind-ra) [inline app/views/privacy-notice.njk:41]: Add  ‘Plan your Shared Parental Leave and Pay’ for ‘Check if you can get Shared Parental Leave and Pay’

File: app/views/privacy-notice.njk [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+            <h2 class="govuk-heading-m">The legal basis for processing your personal data</h2>
+            <p>The legal bases for processing your personal data under (Article 6(1) UK General Data Protection Regulation (GDPR)) are that:</p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>processing is necessary for compliance with a legal obligation to which the controller is subject</li>
+                <li>processing is necessary for a task carried out in the public interest or in the exercise of official authority vested in the controller</li>
+            </ul>
+
+            <h2 class="govuk-heading-m">How DBT processes personal data it receives</h2>
+            <p>Data entered into the ‘Check if you can get Shared Parental Leave and Pay’ service is not stored in any database. It is only held temporarily to enable you to complete your use of the service.</p>
```
Comment (devind-ra) [inline app/views/privacy-notice.njk:52]: Add second paragraph from the document in this section from the PT privacy notice

**Review by devind-ra — COMMENTED** [review]
Looks good to me! Just needs minor edits as there is a separate privacy notice for PT (linked on ticket in the comments) and for ET.

File: app/views/privacy-notice.njk (hunk 1 above)
Comment (Louis-Solirius) [reply app/views/privacy-notice.njk:28]: Ah, you've caught me being lazy! I did just steal it from your PR haha

Comment (devind-ra) [reply app/views/privacy-notice.njk:28]: haha that makes a lot of sense now, it's all good

**Review by lambley — CHANGES_REQUESTED** [review]
Changes needed to text - see comments fromm @devind-ra

**Review by devind-ra — APPROVED** [review]

**Pull request merged**